draft-opsarea-rfc5706bis.md

```diff
@@ -729,29 +706,11 @@ This document does not describe interoperability requirements. As such, it does
    where one understands the network impact (as part of the total cost
    of operations) of that service.
 
-   Getting everybody to agree on a single syntax and an associated
```

Comment:
Can you cut this change out as I have re-written this section from scratch?

Reply:
Reverted it back, @xorrkaz